### PR TITLE
feat: Review's importer test | NPG-8040

### PR DESCRIPTION
**Pull request by Mr-Leshiy**
# Description

This is an initial review's importer test, right now it is not functional because we need to prepare data in ideascale with which this test will operate.
But the general test case already defined.

- Added a review's importer test scenario
- Update dev stage data
- Fixed cat-data-service and event-db tests with the enabled stage data

### CODE REVIEW

File: src/cat-data-service/src/service/mod.rs [hunk 1]
```diff
@@ -145,8 +145,10 @@ async fn track_metrics<T>(req: Request<T>, next: Next<T>) -> impl IntoResponse {
...
     use std::str::FromStr;
 
-    pub fn body_data_json_check(body_data: Vec<u8>, expected_json: serde_json::Value) -> bool {
-        serde_json::Value::from_str(String::from_utf8(body_data).unwrap().as_str()).unwrap()
-            == expected_json
+    pub fn body_data_json_check(body_data: Vec<u8>, expected_json: serde_json::Value) {
+        assert_eq!(
+            serde_json::Value::from_str(String::from_utf8(body_data).unwrap().as_str()).unwrap(),
+            expected_json
+        );
```
Comment (kukkok3) [inline src/cat-data-service/src/service/mod.rs:152]: Are 100% you sure those unwrap will not panic?

File: src/cat-data-service/src/service/v0/fund.rs [hunk 2]
```diff
@@ -57,7 +57,7 @@ mod tests {
             .unwrap();
         let response = app.clone().oneshot(request).await.unwrap();
         assert_eq!(response.status(), StatusCode::OK);
-        assert!(body_data_json_check(
+        body_data_json_check(
             response.into_body().data().await.unwrap().unwrap().to_vec(),
```
Comment (kukkok3) [inline src/cat-data-service/src/service/v0/fund.rs:61]: Same thing regarding this unwrap

File: src/cat-data-service/src/service/v1/event/ballots.rs [hunk 3]
```diff
@@ -66,7 +66,7 @@ mod tests {
             .unwrap();
         let response = app.clone().oneshot(request).await.unwrap();
         assert_eq!(response.status(), StatusCode::OK);
-        assert!(body_data_json_check(
+        body_data_json_check(
             response.into_body().data().await.unwrap().unwrap().to_vec(),
```
Comment (kukkok3) [inline src/cat-data-service/src/service/v1/event/ballots.rs:70]: Same:)

File: src/cat-data-service/src/service/v1/event/mod.rs [hunk 4]
```diff
@@ -105,7 +105,7 @@ mod tests {
             .unwrap();
         let response = app.clone().oneshot(request).await.unwrap();
         assert_eq!(response.status(), StatusCode::OK);
-        assert!(body_data_json_check(
+        body_data_json_check(
             response.into_body().data().await.unwrap().unwrap().to_vec(),
```
Comment (kukkok3) [inline src/cat-data-service/src/service/v1/event/mod.rs:109]: Same here

**Review by stevenj — APPROVED** [review]
LGTM

**Review by kukkok3 — APPROVED** [review]
LGTM

**Pull request merged**